programs/kunit/run

```diff
@@ -11,7 +11,7 @@ if [[ "$group" = "group-00" ]]; then
 	test_array=("bitfield_kunit" "test_bits" "cmdline_kunit" "ext4-inode-test" "test_hash" \
 	            "test_linear_ranges" "list-test" "memcpy_kunit" "dev_addr_lists_test" "resource_kunit" \
 	            "lib_test" "sysctl-test" "time_test" "test_list_sort" "test_sort" "kasan_test" \
-	            "kfence_test" "mptcp_crypto_test" "mptcp_token_test" "rational-test" "slub_kunit" \
+	           "kmsan_test" "kfence_test" "mptcp_crypto_test" "mptcp_token_test" "rational-test" "slub_kunit" \
```

Comment:
why do you still add this, pls check early comments carefully

Reply:
ok